includes/oauth-php/OAuth.php

```diff
@@ -100,17 +100,17 @@ public function check_signature($request, $consumer, $token, $signature) {
     // Avoid a timing leak with a (hopefully) time insensitive compare
     $result = 0;
     for ($i = 0; $i < strlen($signature); $i++) {
-      $result |= ord($built{$i}) ^ ord($signature{$i});
+      $result |= ord($built{$i}) ^ ord($signature[$i]);
```

Comment:
Is `built` a string here? If so, we should change those braces too.

Reply:
Hmmm, yes and that one was still throwing a warning when I had fixed it so I'm not sure what happened. Anyway, fixed in cd59153b1ad589c13a27f429fe8fb6bcbf0d49db